corehq/apps/smsbillables/migrations/0011_remove_like_indexes.py

```diff
@@ -0,0 +1,80 @@
+# encoding: utf-8
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        db.execute("DROP INDEX IF EXISTS smsbillables_smsbillable_direction_like")
+        db.execute("DROP INDEX IF EXISTS smsbillables_smsbillable_domain_like")
+        db.execute("DROP INDEX IF EXISTS smsbillables_smsgatewayfeecriteria_backend_api_id_like")
+        db.execute("DROP INDEX IF EXISTS smsbillables_smsgatewayfeecriteria_backend_instance_like")
+        db.execute("DROP INDEX IF EXISTS smsbillables_smsgatewayfeecriteria_direction_like")
+        db.execute("DROP INDEX IF EXISTS smsbillables_smsgatewayfeecriteria_prefix_like")
+        db.execute("DROP INDEX IF EXISTS smsbillables_smsusagefeecriteria_direction_like")
+        db.execute("DROP INDEX IF EXISTS smsbillables_smsusagefeecriteria_domain_like")
```

Comment:
will defer to @nickpell on the smsbillables ones, but I'm good with dropping the like indexes on the sms_* and smsforms_* tables